### DISCUSSION TITLE
Unique visitors per month?

### DISCUSSION THREAD

**Discussion by raine** [allinurl/goaccess#2230] [Ideas]
When generating report for a longer time period, it would be great if you could interactively change the timeframe for "Unique visitors per day" to be per month or maybe per week as well.

![image](https://user-images.githubusercontent.com/11027/142385107-f8f20d1d-3db5-4123-a4a7-f00801bd3e8d.png)

Thoughts? Or does this already exist?

**Comment by allinurl** [comment]
It doesn't exist yet. Just to be clear, you would like to group the output of the graph by week or month?

**Reply by raine** [reply]
Yeah, that's what I was trying to say, group by is the operation indeed.

**Reply by raine** [reply]
Funny. Playing around with goaccess again, wondering if this feature exists and I find my own question through Google.

**Reply by allinurl** [reply]
#117 will do it. [Stay tuned.](https://x.com/goaccess/)

**Comment by GitRobGit** [comment]
I have the same interest - grouping of unique visitors by month would be very helpful. Has this been implemented? For me on v 1.10.2, grouping is only by day. Or is there an option I should be using? I didn't see anything in the man pages.
